brighthive/convert_descriptor_to_swagger
# Convert list of descriptors to swagger file

## Overview

This will convert a list of BrightHive Data Resource API descriptor files to a swagger spec.

### Notes

Currently this does not support the many to many routing.

## Testing Instructions

 * Run https://github.com/brighthive/descriptor-to-swagger-test
 * Verify the swagger doc it produces is correct.
